### PR TITLE
routing: change adminMode parameter boolean to observable

### PR DESCRIPTION
**Pull request by Garfield-fr**
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- Change boolean to observable.

## How to test?

- Change parameter on routing to check disabled admin actions.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.

### CODE REVIEW

File: projects/ng-core-tester/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -17,15 +17,29 @@
...
-import { Observable, of } from 'rxjs';
+import { ActionStatus } from '@rero/ng-core';
 import { JSONSchema7 } from 'json-schema';
-
+import { Observable, of } from 'rxjs';
 import { HomeComponent } from './home/home.component';
 import { DetailComponent } from './record/document/detail/detail.component';
 import { DocumentComponent } from './record/document/document.component';
-import { ActionStatus } from 'projects/rero/ng-core/src/public-api';
 import { RouteService } from './routes/route.service';
```
Comment (sebastiendeleze) [inline projects/ng-core-tester/src/app/app-routing.module.ts]: re-organize imports

File: projects/rero/ng-core/src/lib/record/detail/detail.component.ts [hunk 2]
```diff
@@ -14,20 +14,18 @@
...
-import { RecordDetailDirective } from './detail.directive';
-import { JsonComponent } from './view/json.component';
-import { RecordService } from '../record.service';
+import { Observable } from 'rxjs';
 import { ActionStatus } from '../action-status';
 import { RecordUiService } from '../record-ui.service';
-import { TranslateService } from '@ngx-translate/core';
+import { RecordService } from '../record.service';
+import { RecordDetailDirective } from './detail.directive';
+import { JsonComponent } from './view/json.component';
```
Comment (sebastiendeleze) [inline projects/rero/ng-core/src/lib/record/detail/detail.component.ts]: organize imports

File: projects/ng-core-tester/src/app/app-routing.module.ts (hunk 1 above)
Comment (Garfield-fr) [reply projects/ng-core-tester/src/app/app-routing.module.ts]: done

File: projects/rero/ng-core/src/lib/record/detail/detail.component.ts (hunk 2 above)
Comment (Garfield-fr) [reply projects/rero/ng-core/src/lib/record/detail/detail.component.ts]: done

**Pull request merged**